xml/security_ldap.xml

```diff
@@ -682,12 +447,21 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
     <command>systemctl</command> commands as described
     in <xref linkend="sec.boot.systemd.basics.services"/>.
    </para>
-  <!-- </sect2> -->
+
+  <sect2 xml:id="sec.security.ldap.server.setup">
+  <title>Installation and Basic Setup</title>
+   <para>Install the 389-ds package:</para>
+  <screen>&prompt.sudo;zypper install 389-ds</screen>
+ <!--next steps:
+     create instance.inf
+     dscreate from-file etc.-->
+  </sect2>
  </sect1>
 <!-- ==================================================================== -->
  <sect1 xml:id="sec.security.ldap.data" os="sles;osuse">
   <title>Manually Administering LDAP Data</title>
-
+<!--taroth 2019-02-25: let's see where to move/integrate the content from this
+    section in the end-->
```

Comment:
I suspect a lot of this will change as 389-ds supplies the dsidm command, which makes the need for ldapsearch/ldapmodify a rare-occurance. A personal goal in 389-ds is so I never have to see ldapmodify again as it's super hard to use and confusing.

Reply:
Added your reply to the remark for the records (and later reference).